.github/workflows/test-sui.yaml

```diff
@@ -110,6 +110,9 @@ jobs:
       - name: Deploy Utils
         run: node sui/deploy-contract deploy Utils
 
+      - name: Deploy VersionControl
+        run: node sui/deploy-contract deploy VersionControl
+
```

Comment:
@jcs47 `Utils` and `VersionControl` contracts need to be deployed before Gateway contract like above